rpm/qtmozembed-qt5.spec

```diff
@@ -30,6 +30,7 @@ BuildRequires:  pkgconfig(systemsettings) >= 0.5.25
 Requires:       xulrunner-qt5 >= %{min_xulrunner_version}
 Requires:       nemo-qml-plugin-systemsettings >= 0.5.25
 Requires:       embedlite-components-qt5 >= 1.22.32
+Requires:       sailfishsilica-qt5 >= 1.1.136
```

Comment:
Please drop this before merging.

Reply:
Thanks for spotting this. I've removed it.